src/shared/ui/SearchForm/SearchForm.js

```diff
@@ -19,13 +19,13 @@ export const SearchForm = ({ initialValue = "", onSearch }) => {
   }
 
   return (
-    <form class="search-form input-element">
-      <input type="text" placeholder="Search...."
+    <form className="search-form input-element">
+      <input type="text" role="txtSearch" placeholder="Search...."
```

Comment:
let's use `data-testid`
e.g `data-testid="search-input"`
`getByTestId("search-input")`

Reply:
changed to search-input